<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File Reader</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Aggregation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="aggregation">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maximum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Median</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Average</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Image Transform:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="transform">
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mirror about Vertical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mirror about Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Transpose</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotate 90°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotate 180°</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotate 270°</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Image Files:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Dark Mode:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="darkMode">
        <property name="currentIndex">
         <number>5</number>
        </property>
        <item>
         <property name="text">
          <string>Median</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Empty Frames</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Average</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maximum</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="selectDark">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Select Dark File</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="image_folder">
        <property name="text">
         <string>Change Image Folder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="image_files">
        <property name="text">
         <string>Select Image Files</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" rowspan="2">
       <widget class="QCheckBox" name="aps_imageseries">
        <property name="text">
         <string>Use APS ImageSeries</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" rowspan="2">
       <widget class="QPushButton" name="read">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Read Files</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Multiframe Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Detector:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="detector">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string>detector_1</string>
          </property>
          <item>
           <property name="text">
            <string>detector_1</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="all_detectors">
          <property name="text">
           <string>Apply Selections to All Detectors</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="file_options">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ContiguousSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>85</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Image File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Empty Frames</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Total Frames</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Omega Min</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Omega Max</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Delta</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Information</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Image Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="img_directory">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dark File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="dark_file">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
